routes/api/project/location/load_balancer.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class CloverApi
+  hash_branch(:project_location_prefix, "load-balancer") do |r|
+    pss = @project.private_subnets_dataset.where(location: @location).all
+    r.get true do
+      Routes::Common::LoadBalancerHelper.new(app: self, request: r, user: @current_user, location: nil, resource: nil).list
+    end
+
+    r.on "id" do
+      r.is String do |lb_id|
+        unless pss.any? { _1.load_balancers.map(&:ubid).include?(lb_id) }
+          response.status = 404
+          r.halt
+        end
+
+        lb = LoadBalancer.from_ubid(lb_id)
+        Routes::Common::LoadBalancerHelper.new(app: self, request: r, user: @current_user, location: nil, resource: lb).get
```

Comment:
We should have delete, attach-vm and detach-vm endoints with id as well.

Reply:
I had no idea we had those APIs, tbh. I have added them.